adadamp/_dist.py

```diff
@@ -447,3 +452,28 @@ def score(self, X, y=None):
         self._meta.update(stat)
         self._meta["score__calls"] += 1
         return acc
+
+class DaskClassifierIncreasingLR(DaskClassifier):
```

Comment:
I will change this to decreasing

Reply:
This class implementation isn't ideal, but it works for now. Give me a week to get this cleaned up.

Reply:
Any progress on this?